app/reducers/surveys.ts

```diff
@@ -55,7 +55,7 @@ export const selectSurveys = createSelector(
 
 export const selectSurveyById = createSelector(
   (state: RootState) => selectSurveys(state),
-  (_: RootState, surveyId: ID) => surveyId,
+  (_: RootState, surveyId?: ID) => surveyId,
```

Comment:
You've changed other surveyId's to `string`. Is it intentional that it is still `ID` here?

Reply:
They should actually be `ID` - I thought they needed to be `string` (like it actually is since it's retrieved from query) to fix a ts error, but setting it as optional was enough. Rewrote it now